{
  "id": "CVE-2019-19911",
  "package_name": "pillow",
  "fix_state": "unknown",
  "cpes": [
    "cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*"
  ],
  "version_constraint": "< 6.2.2",
  "fixed_in_versions": "null",
  "severity": "High",
  "description": "There is a DoS vulnerability in Pillow before 6.2.2 caused by FpxImagePlugin.py calling the range function on an unvalidated 32-bit integer if the number of bands is large. On Windows running 32-bit Python, this results in an OverflowError or MemoryError due to the 2 GB limit. However, on Linux running 64-bit Python this results in the process being terminated by the OOM killer.",
  "urls": [
    "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3DUMIBUYGJRAVJCTFUWBRLVQKOUTVX5P/",
    "https://pillow.readthedocs.io/en/stable/releasenotes/6.2.2.html",
    "https://usn.ubuntu.com/4272-1/",
    "https://www.debian.org/security/2020/dsa-4631"
  ]
}
